api/envoy/http/service_control/config.proto

```diff
@@ -131,8 +131,7 @@ message FilterConfig {
 
   // The prefix added to generated headers
   string generated_header_prefix = 9 [(validate.rules).string = {
-    well_known_regex: HTTP_HEADER_VALUE,
-    strict: false,
+    well_known_regex: HTTP_HEADER_NAME,
```

Comment:
Will header prefix `X-Endpoint-` be regarded as a valid header name?

Reply:
Yes. Our integration tests still pass, so it's definitely valid.